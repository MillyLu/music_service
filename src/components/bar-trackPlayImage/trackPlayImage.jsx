import { useContext } from "react";
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles';


export function TrackPlayImage(){
    const {theme} = useContext(ThemeContext);

    return(
        <Styled.TrackPlayImage theme={theme}>
            <Styled.TrackPlaySvg></Styled.TrackPlaySvg>
        </Styled.TrackPlayImage>
    )
}