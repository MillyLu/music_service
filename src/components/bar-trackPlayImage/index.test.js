/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { TrackPlayImage } from './trackPlayImage';



describe('<TrackPlayImage/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<TrackPlayImage />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });
