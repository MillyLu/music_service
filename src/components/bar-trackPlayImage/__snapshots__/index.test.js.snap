// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackPlayImage/> should render successfully and match snapshot 1`] = `
<styled.div
  theme={0}
>
  <Styled(Component) />
</styled.div>
`;
